### ISSUE TITLE
Issue with installing vibe.d on android for termux

### ISSUE THREAD

**Issue by Fernal73** [vibe-d/eventcore#190]
Linking to said issue and comment:-

https://github.com/termux/termux-packages/issues/7089#issuecomment-873851141

https://github.com/termux/termux-packages/issues/7089

**Comment by s-ludwig** [comment]
This should be fixed with eventcore 0.9.17 and vibe-core 1.18.1 (coincidentally released yesterday).

**Comment by Fernal73** [comment]
That seems to have resolved the error.
```
es/vibe-d-0.9.3/vibe-d
Target is a library. Skipping execution.
```
